packages/create-webstone-app/src/helpers.ts

```diff
@@ -67,34 +61,23 @@ export const installWebApp = async (appDir: string) => {
     // An empty directory means `pnpm init svelte@next` is not asking to overwrite it
     fs.removeSync(`${webAppDir}/.keep`);
 
-    const svelteInitProcess = execa("pnpm", ["init", "svelte@next", "."], {
-      cwd: webAppDir,
-      shell: true,
-      // stdio: "inherit",
-    });
-
-    const waitAndWrite = async (content: string) => {
-      // Thanks to https://github.com/svelte-add/svelte-add/blob/main/projects/create-kit/__init.js
-      await wait(300);
-      svelteInitProcess.stdin?.write(content);
-    };
-
-    // It doesn't look like SvelteKit will get a public API to automate the initialization.
-    // What follows is a hacky, not to mention fragile workaround.
-    // Context: https://github.com/sveltejs/kit/issues/2348 and linked issues / PRs
```

Comment:
As you mentioned, this step is a bit hacky. Thats why I refactored it to use the `svelte-add` approach, where ts, eslint, etc... can be passed in as cli arguments.

The packages get installed automatically.